### ISSUE TITLE
Permanently changed new AirPods Pro 2 Model Number

### ISSUE THREAD

**Issue by xSonus** [LNSSPsd/PodsGrant#59]
Is there any way to revert it? I’m running into a bunch of problems. It’s saying the model number is A2698 when they’re the A2913 model, and my phone is just whacking out when using them. Any solution? They were my only birthday gift yesterday. They were from myself. Super upset about everything, I’d just like to resolve this issue if possible to use them normally. Thank you very much.

**Comment by xSonus** [comment]
Anyone have any ideas? Basically I need to make my AirPods Pro 2 show up as the original model, A2913, instead of the model PodsGrant assigned to them which is A2698. Would really appreciate any and all help!

**Comment by thexYeti** [comment]
For what it's worth, I haven't used podsgrant and my app2 are also showing as model number A2698. You sure it's been changed?

**Comment by rastafaa** [comment]
Your description of issue doesn't make much sense to me, but you can reset the airpods if you want, check the manual

**Comment by xSonus** [comment]
Yes I’m positive. Mine are newer issue. Check the model #’s on Apple. 2698 is first release. Newest is 2913 which is also what’s engraved on the app2’s themselves. Audios all whacked, can’t make calls either. Not sure how to fix. May just have to return to Apple and get a new pair lol
